overlay.d/15rhcos-pcrphase-deps/etc/systemd/system/systemd-pcrphase.service

```diff
@@ -0,0 +1,27 @@
+#  SPDX-License-Identifier: LGPL-2.1-or-later
+#
+#  This file is part of systemd.
+#
+#  systemd is free software; you can redistribute it and/or modify it
+#  under the terms of the GNU Lesser General Public License as published by
+#  the Free Software Foundation; either version 2.1 of the License, or
+#  (at your option) any later version.
+
+# This is a manually copy-pasted unit definition with `After=` section removed.
+# It is because dependency on remote-fs.target causes lockdown in case of any
+# network issues, causing this service to block ability to login to the node.
+# Related issues: OCPBUGS-11124, OCPBUGS-11411
+
+[Unit]
+Description=TPM2 PCR Barrier (User)
+Documentation=man:systemd-pcrphase.service(8)
+Before=systemd-user-sessions.service
+AssertPathExists=!/etc/initrd-release
+ConditionSecurity=tpm2
+ConditionPathExists=/sys/firmware/efi/efivars/StubPcrKernelImage-4a67b082-0a4c-41cf-b6c7-440b29bb8c4f
```

Comment:
What's the significance of this UUID? This looks painful to keep in sync with the RPM.

Reply:
It's just a hardcoded value, seems like a global UUID used by EFI, https://systemd.io/BOOT_LOADER_INTERFACE/